packages/qwik/src/cli/migrate-v2/replace-package.ts

```diff
@@ -9,8 +9,14 @@ function updateFileContent(path: string, content: string) {
   log.info(`"${path}" has been updated`);
 }
 
-export function replacePackage(oldPackageName: string, newPackageName: string): void {
-  replacePackageInDependencies(oldPackageName, newPackageName);
+export function replacePackage(
+  oldPackageName: string,
+  newPackageName: string,
+  skipDependencies = false
+): void {
+  if (skipDependencies) {
```

Comment:
shouldn't it be "not" like `if (!skipDependencies)` ?

Reply:
nice catch, thanks!